src/btree/bt_debug.c

```diff
@@ -794,41 +776,63 @@ __wt_debug_disk(
     WT_DBG *ds, _ds;
     WT_DECL_RET;
     uint32_t flags;
+    bool is_first_flag;
 
     ds = &_ds;
     flags = dump_app_data ? WT_DEBUG_UNREDACT : 0;
+    is_first_flag = true;
     WT_RET(__debug_config(session, ds, ofile, flags));
 
-    WT_ERR(ds->f(ds, "%s page", __wt_page_type_string(dsk->type)));
+    WT_ERR(ds->f(ds, "- %s page\n\t> ", __wt_page_type_string(dsk->type)));
     switch (dsk->type) {
     case WT_PAGE_BLOCK_MANAGER:
         break;
     case WT_PAGE_COL_FIX:
     case WT_PAGE_COL_INT:
     case WT_PAGE_COL_VAR:
-        WT_ERR(ds->f(ds, ", recno %" PRIu64, dsk->recno));
+        WT_ERR(ds->f(ds, "recno: %" PRIu64 " | ", dsk->recno));
```

Comment:
in this particular case, `recno` is the first item printed in the list of metadata associated with a particular case. So I can't have the `|` in the beginning

I.e.
```
...
- column-store internal page
	> recno: 1 | entries: 5 | page_flags: [fast_trunc_update] | generation: 14
...
```
Would be:
```
...
- column-store internal page
	> | recno: 1 | entries: 5 | page_flags: [fast_trunc_update] | generation: 14
...
```

Reply:
In the first case, we could skip it. 
The idea is that if someone wants to append to the metadata, then they should have "|" in the beginning. Instead of going to the last item and appending "|" at the end.

Reply:
I agree with your point but since our metadata list can start with `recno` or `entries` or `datalen` (or possibly something else) it becomes problematic trying to keep track of which item is the first in the metadata list. I chose to do it this way to avoid `|` in the beginning of the list
i.e.
```
> recno: 1 | entries: 5 | dsk_flags: [fast_trunc_update] | generation: 14
```
or
```
 > entries: 32 | dsk_flags: [fast_trunc_update] | generation: 34
```

Reply:
Ahh got it :)